.github/workflows/cla-check.yaml

```diff
@@ -0,0 +1,11 @@
+name: CLA Check
+on:
+  pull_request:
+  workflow_call:
+
+jobs:
+  cla-check:
+    runs-on: ubuntu-20.04
```

Comment:
nit: why not `ubuntu-latest` though?

Reply:
not special reason. I just prefer to avoid the risk of it failing in the future due to some incompatibility with a release that is yet to come